src/reconciler/sync_vectors.py

```
@@ -425,33 +425,31 @@ async def _sync_message_embeddings(
+        with embedding_call_purpose(
+            EmbeddingCallPurpose.MESSAGE_CREATE.value,
+            workspace_name=workspaces.pop() if len(workspaces) == 1 else None,
+            parent_category="reconciliation",
+        ):
+            for emb in embs_needing_embed:
+                try:
+                    new_emb = await embedding_client.simple_batch_embed([emb.content])
+                    freshly_embedded[emb.id] = new_emb[0]
+                    if store_in_postgres:
+                        emb.embedding = new_emb[0]
+                except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)